### PR TITLE
bt-370: Create seeds for badges and hard skills

### LINKED ISSUE #370: Сreate seeds to create all base badges and hard skills
create a list of hard skills and make a seed
 
**Acceptance criteria:**

Should be the following list of badges:

By personality: Doer / Connector / Thinker
By soft skills: Communicative, Collaboration, Problem-solving, Leadership, Creative, Proactive
By BSA score: Project average score, Lectures average score, Communication score, Working with team score, Level of English, Punctuality

Should be the following list of skills:

JavaScript
Python
Java
C++
Ruby
PHP
Swift
Go
Kotlin
HTML/CSS
React.js
Angular
Node.js
SQL Database Management
NoSQL Database Management
Database Design
Cloud Computing
DevOps 
Cybersecurity
Network Administration
Big Data 
Machine Learning
Artificial Intelligence
Web Development
Mobile App Development
API Development
Software Testing
Project Management
UI/UX Design
Front-end Development
Back-end Development
Full-stack Development
Blockchain Development
Data Science
IT Support
Scripting
IT Security Compliance
Automation
Data Analytics
Cloud Security
RESTful APIs
System Administration
Continuous Integration/Continuous Deployment (CI/CD)
Marketing 
Project Management
Product Management
SEO
AWS
Azure
GCP
Slack
Miro
Git
Figma
Microsoft Office Suite
G Suite
Dropbox
Trello
Asana
JIRA
Confluence
Basecamp
Git

### PR DESCRIPTION
**Pull request by Simpsonkk**
I updated bsa badges table. We could not set the same TYPE field for several bsa badges and MAX_SCORE should not be notNullable.
I did seed for bsa badges and hard skills. Please run this script 'seed:dev:run' after refreshing your migration.

### CODE REVIEW

File: backend/src/seeds/bsa-badges-seed.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    [ColumnName.NAME]: badgeName,
+                    [ColumnName.TYPE]: BadgesTypes.SOFT_SKILLS,
+                }),
+            );
+            await trx(TABLE_NAME).insert(softSkillsBadgesMappedSeed);
+        });
+    } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log(`Seeding error: ${error}`);
+    }
```
Comment (MaksymGavatuk) [inline backend/src/seeds/bsa-badges-seed.ts]: Do we need this try catch or maybe seed throw error to console ?

Comment (Simpsonkk) [reply backend/src/seeds/bsa-badges-seed.ts]: seed throw error to console without try catch I thought it was good practice. I removed try catch.

**Review by MaksymGavatuk — APPROVED** [review]
Looks good. Cool job !

**Review by vladjsp — APPROVED** [review]
works as expected.

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**